view/src/main/java/org/xcolab/view/pages/proposals/utils/voting/VoteValidator.java

```diff
@@ -120,11 +135,9 @@ private boolean isRecentVote(ProposalVote otherVote) {
         return otherVoteTime.plusHours(VOTE_RECENCY_THRESHOLD_HOURS).isAfterNow();
     }
 
-    private boolean isNameSimilar(Member otherUser) {
-        return StringUtils.getLevenshteinDistance(member.getFirstName(),
-                otherUser.getFirstName()) < NAME_SIMILARITY_LEVENSHTEIN_THRESHOLD
-                && StringUtils.getLevenshteinDistance(member.getLastName(),
-                otherUser.getLastName()) < NAME_SIMILARITY_LEVENSHTEIN_THRESHOLD;
+    private boolean nameMatches(Member otherUser) {
```

Comment:
1. The method name is confusing.
2. This method does not seem to belong in here. It should probably be either in `Member` or in some `Util`. If latter, then `static` and with two `Member` arguments.